### DISCUSSION TITLE
Internal image links in authors.yml

### DISCUSSION THREAD

**Discussion by MitPitt** [facebook/docusaurus#8664] [Q&A] [answered]
How can I use internal image links in `authors.yml`?
`image_url` only seems to be working with URLs from the web.
I want to use an image from a folder in my project.

Also same question for `image` in blog frontmatter

**Comment by slorber** [comment, marked as answer]
`image` blog frontmatter should support co-localized image assets like this: `image: ./img/social-card.png` (we use this in recent blog posts if you want to take a look)

It's not possible to do the same using `authors.yml` at the moment but I'd like to make it possible in the future. 

In the meantime you can probably put images in `/static/authors/MitPitt.png` and use `image_url: "/authors/MitPitt.png"`
